packages/react-components/eslint-plugin-react-components/package.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "@fluentui/eslint-plugin-react-components",
+  "version": "0.0.1",
+  "description": "ESLint plugin and custom rules for Fluent UI components v9",
+  "private": true,
+  "main": "./lib-commonjs/index.js",
+  "typings": "./dist/index.d.ts",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/microsoft/fluentui"
+  },
+  "license": "MIT",
+  "devDependencies": {
+    "@fluentui/eslint-plugin": "*",
```

Comment:
we don't wanna use in anything new going forward, please remove.

Reply:
removed